conan/tools/cmake/cmakedeps/templates/config.py

```diff
@@ -45,6 +45,12 @@ def template(self):
             message(FATAL_ERROR "The 'CMakeDeps' generator only works with CMake >= 3.15")
         endif()
 
+        if({{ file_name }}_FIND_QUIETLY)
+            set({{ file_name }}_MESSAGE_MODE VERBOSE)
```

Comment:
Should I unset `{{ file_name }}_MESSAGE_MODE` at the end of either/both files?

Reply:
I think it might not be necessary, as the scope is well defined, shouldn't be an issue.